samples/milestones/proposed.json

```diff
@@ -0,0 +1,44 @@
+{
+  "type": "JOUST",
+  "version": "0.0.1-alpha.1",
+  "content": [
+    {
+      "type": "book:id",
+      "code": "GEN"
+    },
+    {
+      "type": "chapter:c",
+      "number": "1",
+      "sid": "GEN 1"
+    },
+    {
+      "type": "para:p",
+      "content": [
+        {
+          "type": "verse:v",
+          "number": "1",
+          "sid": "GEN 1:1"
+        },
+        "the first verse ",
+        {
+          "type": "verse:v",
+          "number": "2",
+          "sid": "GEN 1:2"
+        },
+        "the second verse ",
+        {
+          "type": "verse:v",
+          "number": "3",
+          "sid": "GEN 1:3"
+        },
+        {
+          "type": "ms:qt-s",
```

Comment:
I'm not sure this is right. We seem to have an empty content array, the content after the milestone object, and no end milestone, despite it being in the USFM.

Reply:
I was thinking an empty content array is better than missing content attribute.

Reply:
End milestone was lost due to a bug, shall fix it